Cargo.toml

```diff
@@ -16,7 +16,7 @@ edition = "2021"
 [dependencies]
 regex = ">= 0"
 anyhow = ">= 0"
-clap = "=2"
+clap = { version = "3.0", features = ["derive"] }
```

Comment:
Let's make this `^3` until it breaks. I think that'll be better for catching bugfixes and security efforts.